maintainer/CI/dox_warnings.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
`/usr/bin/env sh` is useless as `/usr/bin/env` is not guaranteed by POSIX (neither is `/bin/sh`, by the way). I would just use `/bin/sh`.

Reply:
Here the idea was to be more portable, as the program `env` reads the user `$PATH`. I put this change in the POSIX bucket for convenience, since most scripts modified in that commit had heterogenous shebangs that needed fixing, although it has nothing to do with POSIX. In fact, `sh` might not even point to the `dash` interpreter.